README.md

```diff
@@ -13,8 +13,8 @@ Colab.
 You may also be interested in the hosted TensorBoard solution at
 [TensorBoard.dev][]. You can use TensorBoard.dev to easily host, track, and
 share your ML experiments for free. For example, [this experiment][] shows a
```

Comment:
This link does not currently include HParams. Will this experiment be updated to include HParams, or does the experiment link need to be changed?

Reply:
Good catch. Done. I've updated with a link to a new experiment.